contrib/spring/src/main/java/io/opencensus/contrib/spring/autoconfig/OpenCensusAutoConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.spring.autoconfig;
+
+import io.opencensus.common.ExperimentalApi;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.Ordered;
+
+/**
+ * {@link org.springframework.boot.autoconfigure.EnableAutoConfiguration Auto-configuration} to
+ * enable tracing using OpenCensus.
+ *
+ * @since 0.19.0
+ */
+@Configuration
+@ComponentScan(basePackages = "io.opencensus")
+@ConditionalOnProperty(value = "spring.opencensus.enabled", matchIfMissing = true)
```

Comment:
You should be careful to use the same property namespaces as spring projects. The spring teams did a bunch of work to consolidate their use of properties to mostly be under `spring.` so that the community could have other areas. I would suggest you make `opencensus` the top level namespace

Reply:
I'll change the name.